wiki/Gameplay/Notelock/de.md

```diff
@@ -0,0 +1,32 @@
+---
+tags:
+  - note lock
+  - jittering notes
+  - shaking notes
```

Comment:
are there any equivalents of these in German, or any other name notelock goes by? if yes, please add them to the list (without replacing the original tags)

Reply:
`Wackelnde Noten`? Maybe also `zitternde Noten`, but that sounds a bit weird.

Reply:
if there's no local term for notelock, no need to force it

Reply:
as far as i know there is no actual local term, these sound a bit too forced